code/_globalvars/configuration.dm

```diff
@@ -1,4 +1,4 @@
-GLOBAL_REAL(config, /datum/configuration)
+GLOBAL_REAL(config, /datum/controller/configuration)

```

Comment:
And repathed it to a controller because they have overrides and vars to help being in the MC panel less snowflake